conda/recipes/dask-cudf/run_test.sh

```diff
@@ -18,11 +18,11 @@ if [ "${ARCH}" = "aarch64" ]; then
 fi
 
 # Install the latest version of dask and distributed
-logger "pip install git+https://github.com/dask/distributed.git@main --upgrade --no-deps"
-pip install "git+https://github.com/dask/distributed.git@main" --upgrade --no-deps
+logger "pip install git+https://github.com/dask/distributed.git@2023.1.1 --upgrade --no-deps"
```

Comment:
Should we update this file to use centralized bash logic that is more like the CI scripts?

```
# Dask & Distributed option to install main(nightly) or `conda-forge` packages.
export INSTALL_DASK_MAIN=0

# Dask version to install when `INSTALL_DASK_MAIN=0`
export DASK_STABLE_VERSION="2023.1.1"
```

Reply:
+1 on this.